packages/cli/src/rpc/modules/eth.ts

```diff
@@ -26,7 +26,7 @@ export class eth {
    */
   constructor(client: PortalNetwork, logger: Debugger) {
     this._client = client
-    this._history = client.networks.get(NetworkId.HistoryNetwork) as HistoryNetwork
```

Comment:
Want to add `eth_chainId` to this since we're now using `chainId`?  This would also fix a pain point trying to integrate with ponder.